src/middleWares/auth.js

```diff
@@ -0,0 +1,21 @@
+import jwt from "jsonwebtoken";
+
+const verifyToken = (req, res, next) => {
+  const authHeader = req.headers.authorization;
+
+  if (authHeader) {
+    const token = authHeader.split(" ")[1];
+
+    jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, (err, user) => {
+      if (err) {
+        return res.sendStatus(403);
```

Comment:
401 would be better than 403 because 403 means you are forbidden to access the resource and 401 means you are unauthorized to access